cmake/Modules/LocalDav1d.cmake

```diff
@@ -8,8 +8,8 @@ function(avif_build_local_dav1d)
     else()
         message(STATUS "libavif(AVIF_CODEC_DAV1D=LOCAL): ext/dav1d not found, fetching")
         set(source_dir "${FETCHCONTENT_BASE_DIR}/dav1d-src")
-        list(APPEND download_step_args GIT_REPOSITORY https://code.videolan.org/videolan/dav1d.git GIT_TAG ${AVIF_DAV1D_TAG}
-             GIT_SHALLOW ON
+        list(APPEND download_step_args GIT_REPOSITORY https://github.com/videolan/dav1d.git GIT_TAG ${AVIF_DAV1D_TAG} GIT_SHALLOW
```

Comment:
Is dav1d's master repository often down?

Reply:
it was for several hours when I created that PR. It was never down before that. We can wait and see if it happens again.